modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -170,19 +171,19 @@ if the case is not respected.
 .Follow this procedure to create a new mount point based on WND
 
 . Enter the ownCloud mount point for your new WND share. This _must not_ be an existing folder.
-. Select your authentication method. See xref:enterprise/external_storage/enterprise_only_auth.adoc[Enterprise-Only Authentication Options] for complete information on the five available authentication methods.
+. Select your authentication method. See xref:enterprise/external_storage/enterprise_only_auth.adoc[Enterprise-Only Authentication Options] for complete information on the available authentication methods.
 +
 .Figure 2. WND mountpoint and authorization credentials
 image:enterprise/external_storage/windows_network_drive/wnd-2.png[WND mountpoint and authorization credentials, width=60%]
 . Enter the address of the server that contains the WND share.
-. The Windows share name.
-. The root folder of the share. This is can be the subfolder name, or the `$user` variable for the user's home directory. Note that the LDAP `Internal Username Attribute` must be set to the `samaccountname` for either the share or the root to work, and the user's home directory needs to match the `samaccountname`.
+. The share name provided by Windows .
```

Comment:
```suggestion
. The share name provided by Windows or Samba.
```

Suggested Note: Omit any leading `/` or `\` on the share name. A slash or backslash there may lead to odd behaviour.